lib/src/statsig_options.dart

```diff
@@ -5,5 +5,33 @@ class StatsigOptions {
   /// How long (in seconds) the Statsig client waits for the initial network request. Defaults to 3 seconds
   int initTimeout;
 
-  StatsigOptions([this.api, this.initTimeout = 3]);
+  StatsigEnvironment? environment;
+
+  StatsigOptions({this.api, this.initTimeout = 3, this.environment});
+}
+
+enum StatsigEnvironment {
+  development,
+  staging,
+  production,
+}
+
+extension ToJson on StatsigEnvironment {
+  Map<String, dynamic> toJson() {
+    late final String environmentName;
+    switch (this) {
```

Comment:
code golf: String environmentName = this.toString().replaceAll("StatsigEnvironment.", "");